src/tseda/datastore.py

```diff
@@ -252,14 +252,17 @@ class IndividualsTable(Viewer):
     sample_sets_table = param.ClassSelector(class_=SampleSetsTable)
 
     columns = [
-        "name",
+        "color",
         "population",
         "sample_set_id",
-        "selected",
+        "name_sample",
```

Comment:
The information in this column refers to the population, so maybe call the variable `name_population`?

Reply:
Or maybe `name_sample_set` (`sample_set_name`)?